style.css

```diff
@@ -11,15 +11,26 @@ p {
 
 body {
   background-color: white;
-  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    "Open Sans",
+    "Helvetica Neue",
+    sans-serif;
   color: black;
   font-size: 17px;
   padding: 15px 10px;
   text-align: center;
 }
 
 #container {
-  background-color: #eae8e8;
+  background-color: #eee;
   border-radius: 5px;
 }
```

Comment:
Your work is incomplete always make sure that all your related changes are included in your pull request